### PR TITLE
feat(route): 添加北京邮电大学教务处的路由

### PR DESCRIPTION
**Pull request by Yoruet**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bupt/jwc/tzgg
/bupt/jwc/xwzx
```

## New RSS Route Checklist / 新 RSS 路由检查表

- [ ] New Route / 新的路由
- [ ] Follows [[Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard)](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [[路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
- [ ] Full text / 全文获取
- [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
- [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [[Date and time](https://docs.rsshub.app/joinus/advanced/pub-date)](https://docs.rsshub.app/joinus/advanced/pub-date) / [[日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
- [ ] Parsed / 可以解析
- [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Since both routes share the same code, it would be better to combine them

File: lib/routes/bupt/jwc.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    maintainers: ['Yoruet'],
+    handler,
+    url: 'https://jwc.bupt.edu.cn/',
+};
+
+async function handler(ctx: Context) {
+    const type = ctx.req.param('type'); // 默认类型为通知公告
+    if (!type) {
+        type = 'tzgg';
+    }
```
Comment (TonyRL) [inline lib/routes/bupt/jwc.ts]: You can't reassign a `const`

**Comment by Yoruet** [comment]
> Since both routes share the same code, it would be better to combine them

I have combined these two routes

File: lib/routes/bupt/jwc.ts (hunk 1 above)
Comment (Yoruet) [reply lib/routes/bupt/jwc.ts]: This bug has been fixed

**Review by TonyRL — CHANGES_REQUESTED** [review]
> Please double-check the selectors  https://github.com/DIYgod/RSSHub/actions/runs/10851721535/job/30116108359#step:13:34

https://github.com/DIYgod/RSSHub/pull/16729#pullrequestreview-2303430080

Using `/bupt/jwc/tzgg` will cause RSSHub visits `https://jwc.bupt.edu.cn/` **143 times**.

**Comment by Yoruet** [comment]
> > Please double-check the selectors  https://github.com/DIYgod/RSSHub/actions/runs/10851721535/job/30116108359#step:13:34
> 
> [#16729 (review)](https://github.com/DIYgod/RSSHub/pull/16729#pullrequestreview-2303430080)
> 
> Using `/bupt/jwc/tzgg` will cause RSSHub visits `https://jwc.bupt.edu.cn/` **143 times**.

Sorry I missed this suggestion before, it has been fixed now.

**Pull request merged**